controller/internal/enforcer/nfqdatapath/datapath_tcp.go

```diff
@@ -384,6 +384,9 @@ func (d *Datapath) processApplicationSynAckPacket(tcpPacket *packet.Packet, cont
 			zap.L().Debug("Failed to remove cache entries")
 		}
 
+		if err := d.ignoreFlow(tcpPacket, tcpPacket.WindowsMetadata); err != nil {
+			zap.L().Error("Failed to ignore flow", zap.Error(err))
+		}
 		if err := d.conntrack.UpdateApplicationFlowMark(
```

Comment:
Seems to me that we don't need to pass both tcpPacket and tcpPacket.WindowsMedata , since tcpPacket has that already.

Reply:
you are right. fixed.